PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Description
```

Comment:
This file should be removed. Our templates live in `.github/PULL_REQUEST_TEMPLATE.md`. They either live in one of those locations, and we picked `.github`